src/components/HelpNavbar/index.jsx

```diff
@@ -1,18 +1,22 @@
 import React from "react";
 import { Navbar, Container, Col } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
 import phoneIcon from "../../images/icons/phone.svg";
 import "./index.scss";
 
-const HelpNavbar = () => (
-  <Navbar className="helpNavbar">
-    <Container>
-      <Col>
-        <p>Immediate helpline:</p>
-        <img src={phoneIcon} alt="Phone icon" />
-        <p>+1-340-5558-235</p>
-      </Col>
-    </Container>
-  </Navbar>
-);
+const HelpNavbar = () => {
+  const { t } = useTranslation();
+  return (
+    <Navbar className="helpNavbar">
+      <Container>
+        <Col>
+          <p>{t("helpNavbar.title")}</p>
+          <img src={phoneIcon} alt="Phone icon" />
+          <a href="tel:+1-340-5558-235">{t("helpNavbar.phoneNum")}</a>
```

Comment:
and the phone number is copied again here, so thats 3 places you need to change if the phone number changes.

it should be one.

Reply:
done